### PR TITLE
Release 3.7.1 fixes

### PR DESCRIPTION
**Pull request by sachinsahu731**
All accessibility issue fix PR

### CODE REVIEW

File: src/app/client/src/app/app.component.html [hunk 1]
```diff
@@ -5,7 +5,9 @@
     <div class="ui container">
       <div class="d-flex flex-ai-center flex-jc-flex-end">
         <ul class="d-flex flex-ai-center fsmall sb-color-white mb-0 computer only">
-          <li class="mb-0" *ngIf="!hideHeaderNFooter" (click)="skipToMainContent()">Skip to main content</li><li *ngIf="!hideHeaderNFooter" class="mx-4 mb-0">|</li>
+          <li class="mb-0" *ngIf="!hideHeaderNFooter">
+            <a (click)="skipToMainContent()" class="cursor-pointer sb-color-white skip-to-main-content">Skip to Main Content</a>
```
Comment (rajeevsathish) [inline src/app/client/src/app/app.component.html:9]: use the labels

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/app.component.html (hunk 1 above)
Comment (sachinsahu731) [reply src/app/client/src/app/app.component.html]: @rajeevsathish  Please merge the PR

**Review by sri-pusuluri — APPROVED** [review]

File: src/app/client/src/app/modules/player-helper/components/content-rating/content-rating.component.html [hunk 2]
```diff
@@ -32,9 +32,9 @@
             <input type="checkbox" role="checkbox" [(ngModel)]="feedback.checked" (click)="changeOptions(feedback)">
             <label [class.options-content-checked]="feedback.checked">{{feedback['value']}}</label>
           </div> -->
-          <div class="sb-checkbox sb-checkbox-primary sb-checkbox-rightaligned" *ngFor="let feedback of feedbackObj[contentRating]['options']">
-            <input type="checkbox" id="{{feedback['value']}}" [(ngModel)]="feedback.checked" (click)="changeOptions(feedback)">
-            <label for="{{feedback['value']}}" class="fsmall mb-16">{{feedback['value']}}</label>
+          <div class="sb-checkbox sb-checkbox-primary sb-checkbox-rightaligned" *ngFor="let feedback of feedbackObj[contentRating]['options'];index as i">
+            <input type="checkbox" attr.aria-labelledby="feedbackRating{{i}}" id="feedbackRating{{i}}" [(ngModel)]="feedback.checked" (click)="changeOptions(feedback)">
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/player-helper/components/content-rating/content-rating.component.html:36]: what is i and value why is it diffenrent?

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/player-helper/components/content-rating/content-rating.component.html (hunk 2 above)
Comment (sachinsahu731) [reply src/app/client/src/app/modules/player-helper/components/content-rating/content-rating.component.html:36]: Screenshot (113)

The html id come with string and space,  That is invalid id 's value must not contain whitespace
@rajeevsathish

File: src/app/client/src/app/app.component.html (hunk 1 above)
Comment (sri-pusuluri) [reply src/app/client/src/app/app.component.html]: Sirji.. sree told will do this in 3.8

File: src/app/client/src/app/modules/groups/components/add-member/add-member.component.html [hunk 3]
```diff
@@ -23,7 +23,7 @@
               </label>
               <div class="d-flex flex-jc-space-between flex-dc">
                 <div class="w-100 relative">
-                  <input type="id" name="id" class="sb-form-control" [ngClass]="{'is-invalid': isInvalidUser}" placeholder="{{resourceService?.frmelmnts?.lbl?.enterInstanceId | interpolate:'{instance}': instance}}" [(ngModel)]="memberId" (ngModelChange)="reset()">
+                  <input type="id" name="id" aria-label="diksha id" class="sb-form-control" [ngClass]="{'is-invalid': isInvalidUser}" placeholder="{{resourceService?.frmelmnts?.lbl?.enterInstanceId | interpolate:'{instance}': instance}}" [(ngModel)]="memberId" (ngModelChange)="reset()">
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/groups/components/add-member/add-member.component.html:26]: you cannot have a mention of diksha in sunbird repo please remove this and make it instance

File: src/app/client/src/app/app.component.html [hunk 4]
```diff
@@ -3,24 +3,45 @@
...
+        </div>
+        <div class="d-flex flex-ai-center head-tail-right-side">
+          <div class="pull-right dk-top-head-ul">
+            <ul>
+              <li class="mb-0" *ngIf="!hideHeaderNFooter"><a href="javascript:void(0);" title="Skip to Main Content" (click)="skipToMainContent()" class="cursor-pointer"> {{resourceService?.frmelmnts?.lbl?.skipToMainContent}}</a></li>
+              <li *ngIf="!hideHeaderNFooter" class="mx-4 mb-0">|</li>
+              <li class="screen-reader-link"><a href="/screenreaderaccess.html" class=""
+                  title="Screen Reader Access">{{resourceService?.frmelmnts?.lbl?.screenReaderAccess}}</a></li>
+              <li class="mx-4">|</li>
+              <ng-container *ngIf="userService.loggedIn">
```
Comment (rajeevsathish) [inline src/app/client/src/app/app.component.html:17]: the if and else can go in one single li only

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

**Review by sri-pusuluri — APPROVED** [review]

File: src/app/client/src/app/app.component.html (hunk 4 above)
Comment (sri-pusuluri) [reply src/app/client/src/app/app.component.html]: please check

**Review by rajeevsathish — APPROVED** [review]

**Pull request merged**